Makefile

```diff
@@ -91,10 +91,14 @@ define REDIS8_CONF
 daemonize yes
 protected-mode no
 port 6386
+requirepass foobared
```

Comment:
Test requires instance at 6386 port to be password-less. Either create a new instance or create a new one.

Reply:
"a new instance or create a new one"? what is the different between "new instance" and "new one"?

Reply:
Oops!

"Either use password-less instance or create a new one."